src/graphics/shader.js

```diff
@@ -19,7 +19,7 @@ Object.assign(pc, function () {
      * @param {String} definition.vshader Vertex shader source (GLSL code).
      * @param {String} definition.fshader Fragment shader source (GLSL code).
      * @param {Boolean} definition.useTransformFeedback Specifies that this shader outputs post-VS data to a buffer
-     * @param {Object} precache Triggers imediate link.
```

Comment:
imediate -> immediate

Reply:
This line is removed in this PR, as there is no such argument.